packages/dappmanager/src/daemons/ethMultiClient/index.ts

```diff
@@ -150,7 +151,10 @@ export function startEthMultiClientDaemon(signal: AbortSignal): void {
         if (next.status === "INSTALLED") {
           // If status switched to "INSTALLED", map to fullnode.dappnode
           // Must be done here in case the package is already installed
+          // 1. Domain for BIND package
           db.fullnodeDomainTarget.set(ethClientData[target].dnpName);
+          // 2. Mapping for docker BIND
+          await migrateEthClientFullNode(ethClientData[target].dnpName, true);
```

Comment:
Do you want to remove or add here? Very hard to tell with the `true` value. See comment above about splitting the functions